torch_frame/data/multi_nested_tensor.py

```diff
@@ -275,6 +275,27 @@ def _single_index_select(self, index: int, dim: int) -> MultiNestedTensor:
         else:
             raise RuntimeError(f"Unsupported dim={dim} for index_select.")
 
+    def fillna_col(
+        self,
+        col_index: int,
+        fill_value: Union[int, float, Tensor],
+    ) -> None:
+        start_idx = torch.arange(
+            col_index,
+            self.num_rows * self.num_cols,
+            self.num_cols,
+            device=self.device,
+        )
+        diff = self.offset[start_idx + 1] - self.offset[start_idx]
+        batch, arange = _batched_arange(diff)
+        # Compute values
+        values_col = self.values[self.offset[start_idx][batch] + arange]
```

Comment:
nit: cache `self.offset[start_idx][batch] + arange` for efficiency and better readabiilty. 
```python
values_index = self.offset[start_idx][batch] + arange
values_col = self.values[values_index]
....
self.values[values_index] = values_col
```